Hi3Helper.Core/Lang/en.json

```diff
@@ -8,25 +8,25 @@
     "Title1": "Welcome",
     "Title2": "to Collapse Launcher",
     "Subtitle1": "Hi!",
-    "Subtitle2": "Seems like it's your first time to use this launcher.",
-    "Subtitle3": "Anyway, thank you for trying out this project and to set everything up, please choose where will you be placing this launcher's data into your computer.",
-    "Subtitle4_1": "To Start, click on",
-    "Subtitle4_2": "Locate folder",
-    "Subtitle4_3": "button below to choose where do you want to place the game on.",
-    "FolderInsufficientPermission": "Permission Denied! Choose another folder!",
-    "FolderNotSelected": "Folder hasn't been choosen!",
+    "Subtitle2": "Seems like it's your first time using this launcher.",
+    "Subtitle3": "Thank you for trying out this project! To set everything up, please choose where you want the launcher to install itself.",
```

Comment:
IMO `install itself` should be changed to `install its data` or `put its data`, as the launcher itself is already installed either via usual installer (to Program Files) or in the extracted folder (on Portable version)

Reply:
We can also say `store game data` as well, since I believe that is the reason for this prompt being there in the first place?

Reply:
IIRC, that dialog is for asking where user wants CollapseData folder is, where launcher metadata, background, and logs are. So, its more fitting for it to be called the launcher data or "its data" in this context

Reply:
`to install itself` => `to store its data`, to not repeat `the launcher` twice in the same sentence. Or, we can also do `where you want Collapse to store the launcher data` since the name of the launcher is Collapse. For now I kept it at `to store the launcher data` but let me know if you prefer the other options.